parsec/interfaces/ptg/ptg-compiler/jdf_unparse.c

```diff
@@ -234,7 +234,7 @@ static int jdf_call_unparse(const jdf_call_t *call, FILE *out)
 static int jdf_datatransfer_type_unparse(jdf_datatransfer_type_t dt, FILE *out)
 {
     int err = 0;
-    char start[1] = "[";
+    char start[2] = "[";
```

Comment:
? what happened to `start[1]`?

Reply:
you did that so you tell me :) Looks like we need the extra space for the 0 terminator.

Reply:
Yes, I misread the code. But you're right, we need the terminator for the string,